web/public/mgmt/config.yml

```diff
@@ -2867,6 +2867,34 @@ collections:
           options: ["above-opportunities", "below-opportunities"],
         }
 
+  - name: "quick-actions-label"
+    label: "🟨 QUICK ACTIONS"
+    folder: "content/src"
+    fields:
+      - { name: "" }
+
+  - name: "quick-actions"
+    label: "Quick Actions"
+    folder: "content/src/quick-actions"
+    summary: "{{filename}} - {{name}}"
+    identifier_field: "{{filename}}"
+    slug: "{{fields.filename}}"
+    delete: true
+    sortable_fields: ["name", "filename"]
+    create: true
+    editor:
+      preview: true
+    fields:
+      - { label: "Internal Notes", name: "notesMd", widget: "markdown", required: false}
+      - { label: "Name/Title", name: "name", widget: "string" }
+      - { label: "Filename", name: "filename", widget: "nospace", required: false }
+      - { label: "Id", name: "id", widget: "string" }
+      - { label: "Url Slug", name: "urlSlug", widget: "nospace" }
+      - { label: "Content", name: "content", widget: "markdown" }
```

Comment:
[pebble] The main body of the markdown file needs to be marked as `body` for it to work - see other configurations in this file

Reply:
marked as body